content/zh/docs/concepts/workloads/controllers/replicationcontroller.md

```diff
@@ -0,0 +1,552 @@
+<!--
+reviewers:
```

Comment:
This 'reviewers' section can be removed in localized version because the mentioned reviewers may and may not speak this language.